subsys/mpsl/Kconfig

```diff
@@ -25,6 +25,15 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+config MPSL_SCHEDULER_INIT
+	bool "Initialize MPSL scheduler"
+	default y
+	help
+	  Initialize MPSL scheduler. MPSL scheduler is needed when using:
+	    - Radio scheduler used for Bluetooth LE and/or 802.15.4
+	    - Timeslot
+	    - Radio notifications
```

Comment:
Is there any sense in listing radio notifications here?
You cannot have radio notifications unless you have some radio activity, i.e timeslot, bluetooth or 802.15.4.

SoftDevice controller should have a select on this option, same as it has for MPSL.

Reply:
I'd rather not remove it. It is mentioned in the MPSL doc as a key feature